configs/ulissigroup_h/hpo_cgcnn.yml

```diff
@@ -0,0 +1,46 @@
+includes:
+- /global/homes/b/bwood/machine_learning/open_catalyst_project/baselines/configs/ulissigroup_h/base.yml
```

Comment:
Same comment; would it be possible to make this a relative path?

Reply:
Going to try and remove the relative paths